init.el

```diff
@@ -64,6 +64,12 @@ by Prelude.")
   "This folder stores all the automatically generated save/history-files.")
 (defvar prelude-modules-file (expand-file-name "prelude-modules.el" prelude-dir)
   "This files contains a list of modules that will be loaded by Prelude.")
+(defvar prelude-package-archives nil
```

Comment:
`package-pinned-packages` gets overwritten when `package` is required in prelude-packages. I wanted to load the packages we are pinning and extra repos as part of the config in init before that part of core is even run. If you would prefer me load the config inside prelude-packages, I would be happy to make the changes.